app/views/content/returning-to-teaching.md

```diff
@@ -52,6 +52,10 @@ youtube_video:
   return-to-teaching-advisers-video:
     id: 2NrLm_XId4k
     title: A video about what Return to Teaching Advisers do
+inset_text:
+  international-content:
+    text: If you qualified as a teacher outside the UK, you must have English qualified teacher status (QTS) to get an adviser. <a href="tel:08003892500">Call us on 0800 389 2500 to check if you're eligible</a>.
```

Comment:
```suggestion
    text: If you qualified as a teacher outside the UK, you must have English qualified teacher status (QTS) to get an adviser. <a href="tel:08003892500">Call us on 0800 389 2500 to check if you're eligible for an adviser</a>.
```
@EllieNodder I appreciate my suggestion is a bit repetitive (and could be improved!) but the link text doesn't quite give enough context - could be inferred as if you are eligible to teach if you were skipping through links using a screen reader?